server/cluster/cluster.go

```diff
@@ -338,6 +345,38 @@ func (c *RaftCluster) runBackgroundJobs(interval time.Duration) {
 	}
 }
 
+func (c *RaftCluster) processRegionHeartbeatStats() {
+	defer logutil.LogPanic()
+	defer c.wg.Done()
+
+	interval := maxProcessStatsInterval
+	failpoint.Inject("highFrequencyProcessStats", func() {
+		interval = 100 * time.Microsecond
+	})
+
+	ticker := time.NewTicker(interval)
+	defer ticker.Stop()
+	pendingStatsTasks := make([]*statsTask, 0, maxPendingStatsTasks)
+	for {
+		select {
+		case <-c.ctx.Done():
+			log.Info("process region heartbeat job has been stopped")
+			return
+		case tasks := <-c.statsTaskCh:
+			pendingStatsTasks = append(pendingStatsTasks, tasks...)
+			if len(pendingStatsTasks) < maxPendingStatsTasks {
+				continue
```

Comment:
According to the [Go spec](https://go.dev/ref/spec#Continue_statements):

> A "continue" statement begins the next iteration of the innermost "for" loop at its post statement. The "for" loop must be within the same function.

So the `continue` here will continue to the next for loop if the length of `pendingStatsTasks` is smaller than `maxPendingStatsTasks`?